tests/kubernetes-base-addons/kuttl-test.yaml

```diff
@@ -1,21 +1,7 @@
----
```

Comment:
Unless these are required for this test, let's do this in a different PR so there can be discussion around the merits of these changes.

Reply:
removed